scripts/install.js

```diff
@@ -236,6 +262,11 @@ function bootstrap(cb) {
 			});
 		},
 		function (callback) {
+			// For some reason, when using the --prefix flag, an etc folder is added.
```

Comment:
> global configuration file (defaults to $PREFIX/etc/npmrc; configurable via CLI option --globalconfig or environment variable $NPM_CONF_GLOBALCONFIG)

From what I can see from the NPM history, it creates it preemptively: https://github.com/npm/npm/pull/7249